Fray c/Program.cs

```diff
@@ -0,0 +1,222 @@
+
+            invoiceTemp1.InvoiceId = 1;
+            invoiceTemp1.ProductId = 3;
+            invoiceTemp1.ProductQuantity = 2;
+            invoiceTemp1.ProductPrice = 12;
+            invoiceTemp1.CustomerId = 10;
+            invoiceTemp1.SellerId = 1;
+            invoiceTemp1.RecordDate = DateTime.Now;
+
+            invoiceTemp2.InvoiceId = 2;
+            invoiceTemp2.ProductId = 1;
+            invoiceTemp2.ProductQuantity = 3;
+            invoiceTemp2.ProductPrice = 10;
+            invoiceTemp2.CustomerId = 11;
+            invoiceTemp2.SellerId = 1;
+            invoiceTemp2.RecordDate = DateTime.Now;
+
+            invoiceTemp3.InvoiceId = 3;
+            invoiceTemp3.ProductId = 5;
+            invoiceTemp3.ProductQuantity = 3;
+            invoiceTemp3.ProductPrice = 16;
+            invoiceTemp3.CustomerId = 12;
+            invoiceTemp3.SellerId = 1;
+            invoiceTemp3.RecordDate = DateTime.Now;
+
+            invoiceTemp4.InvoiceId = 4;
+            invoiceTemp4.ProductId = 7;
+            invoiceTemp4.ProductQuantity = 3;
+            invoiceTemp4.ProductPrice = 5;
+            invoiceTemp4.CustomerId = 13;
+            invoiceTemp4.SellerId = 1;
+            invoiceTemp4.RecordDate = DateTime.Now;
+
+            invoiceTemp5.InvoiceId = 5;
+            invoiceTemp5.ProductId = 3;
+            invoiceTemp5.ProductQuantity = 7;
+            invoiceTemp5.ProductPrice = 12;
+            invoiceTemp5.CustomerId = 11;
+            invoiceTemp5.SellerId = 1;
+            invoiceTemp5.RecordDate = DateTime.Now;
+
+            invoiceList.Add(invoiceTemp1);
+            invoiceList.Add(invoiceTemp2);
+            invoiceList.Add(invoiceTemp3);
+            invoiceList.Add(invoiceTemp4);
+            invoiceList.Add(invoiceTemp5);
+
+            return invoiceList;
+        }
+        static void Main(string[] args)
+        {
+            List<Customer> customerList = new List<Customer>();
+            ProductList productList = new ProductList();
+            List<Product> productMinStockList = new List<Product>();
+            List<Invoice> invoiceList = new List<Invoice>();
+            
+            List<Customer> customer55List = new List<Customer>();
+            List<Customer> customerBirthDayList = new List<Customer>();
+
+            customerList = CustomerList();
+            productList = ProductList();
+            invoiceList = InvoiceList();
+
+            Seller sellerTemp = new Seller();
+            sellerTemp.SellerId = 1;
+            sellerTemp.Name = "Carlos Bodger";
+
+            float SalesTotal = 0;
+            foreach(var invoice in invoiceList)
+            {
+                Customer customer55 = new Customer();
+                Customer customerBirthDay = new Customer();
+                int customerId = invoice.CustomerId;
+
+                SalesTotal+= invoice.ProductPrice*invoice.ProductQuantity;
+
+                customer55 = customerList.Find(x => x.CustomerId == customerId && x.Age > 55);

```

Comment:
"x =>..."
this is lambda, good one.

Reply:
but you must to use a better name for your variable, it could be "customer'

Reply:
Name resolved